### PR TITLE
Remove live migration from documentation

### PR DESCRIPTION
**Pull request by rmohr**
In the following release, we will temporarily remove live-migrations.
Let's make sure that the user-guide reflects that.

Signed-off-by: Roman Mohr <rmohr@redhat.com>

### CODE REVIEW

File: feature-matrix.md [hunk 1]
```diff
@@ -18,7 +18,6 @@ The following table lists the available resource types.
 | Resource Type | Available since | Required Kubernetes and kubectl Version |
 | --- | --- | --- |
 | Virtual Machine | v0.1.0 | 1.6 |
-| Migration | v0.1.0 | 1.6 |
```
Comment (fabiand) [inline feature-matrix.md]: Let's do the following:

1. Please opne a bug to reintroduce live migration
2. Stroke this entry and link it to the bug

Comment (rmohr) [reply feature-matrix.md]: Done

Comment (fabiand) [reply feature-matrix.md]: I'm still missing that the entrey in the table above is stroked (~...~) and a link to the bug.

Comment (rmohr) [reply feature-matrix.md]: It is below the feature matrix.

Comment (rmohr) [reply feature-matrix.md]: oh wrong section

Comment (rmohr) [reply feature-matrix.md]: done

Comment (fabiand) [reply feature-matrix.md]: thanks

**Pull request merged**